src/os_posix/os_fs.c

```diff
@@ -406,6 +414,9 @@ __posix_file_read(
     session = (WT_SESSION_IMPL *)wt_session;
     pfh = (WT_FILE_HANDLE_POSIX *)file_handle;
 
+    __wt_verbose(session, WT_VERB_READ, "read: %s, fd=%d, offset=%" PRId64 ", len=%" PRIu64 "\n",
+      file_handle->name, pfh->fd, offset, (uint64_t)len);
+
```

Comment:
There's no reason to cast `len`, you can use `WT_SIZET_FMT`.
Printing a `wt_off_t` does require a cast.

Only flagging this once, there are a few more instances.